packages/network-order-controller/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "rootDir": "../.."
+  },
+  "references": [
+    {
+      "path": "../../packages/base-controller"
+    },
+    {
+      "path": "../../packages/network-controller"
+    },
+    {
+      "path": "../../packages/multichain-network-controller"
+    }
+  ],
+  "include": ["../../types", "./src", "tests/NetworkOrderController.test.ts"]
```

Nit: if you opt in for the suggestion to move `NetworkOrderController.test.ts` to test folder this can be dropped
```suggestion
  "include": ["../../types", "./src"]